stdlib.go

```diff
@@ -1,1247 +1,7 @@
-	"# `DIRENV_EXTRA_VIMRC` environment variable.\n" +
-	"#\n" +
-	"# This variable is understood by the direnv/direnv.vim extension. When found,\n" +
-	"# it will source it after opening files in the directory.\n" +
-	"use_vim() {\n" +
-	"  local extra_vimrc=${1:-.vimrc.local}\n" +
-	"  path_add DIRENV_EXTRA_VIMRC \"$extra_vimrc\"\n" +
-	"}\n" +
-	"\n" +
-	"# Usage: direnv_version <version_at_least>\n" +
-	"#\n" +
-	"# Checks that the direnv version is at least old as <version_at_least>.\n" +
-	"direnv_version() {\n" +
-	"  \"$direnv\" version \"$@\"\n" +
-	"}\n" +
-	"\n" +
-	"# Usage: on_git_branch [<branch_name>]\n" +
-	"#\n" +
-	"# Returns 0 if within a git repository with given `branch_name`. If no branch\n" +
-	"# name is provided, then returns 0 when within _any_ branch. Requires the git\n" +
-	"# command to be installed. Returns 1 otherwise.\n" +
-	"#\n" +
-	"# When a branch is specified, then `.git/HEAD` is watched so that\n" +
-	"# entering/exiting a branch triggers a reload.\n" +
-	"#\n" +
-	"# Example (.envrc):\n" +
-	"#\n" +
-	"#    if on_git_branch child_changes; then\n" +
-	"#      export MERGE_BASE_BRANCH=parent_changes\n" +
-	"#    fi\n" +
-	"#\n" +
-	"#    if on_git_branch; then\n" +
-	"#      echo \"Thanks for contributing to a GitHub project!\"\n" +
-	"#    fi\n" +
-	"on_git_branch() {\n" +
-	"  local git_dir\n" +
-	"  if ! has git; then\n" +
-	"    log_error \"on_git_branch needs git, which could not be found on your system\"\n" +
-	"    return 1\n" +
-	"  elif ! git_dir=$(git rev-parse --absolute-git-dir 2> /dev/null); then\n" +
-	"    log_error \"on_git_branch could not locate the .git directory corresponding to the current working directory\"\n" +
-	"    return 1\n" +
-	"  elif [ -z \"$1\" ]; then\n" +
-	"    return 0\n" +
-	"  fi\n" +
-	"  watch_file \"$git_dir/HEAD\"\n" +
-	"  [ \"$(git branch --show-current)\" = \"$1\" ]\n" +
-	"}\n" +
-	"\n" +
-	"# Usage: __main__ <cmd> [...<args>]\n" +
-	"#\n" +
-	"# Used by rc.go\n" +
-	"__main__() {\n" +
-	"  # reserve stdout for dumping\n" +
-	"  exec 3>&1\n" +
-	"  exec 1>&2\n" +
-	"\n" +
-	"  __dump_at_exit() {\n" +
-	"    local ret=$?\n" +
-	"    \"$direnv\" dump json \"\" >&3\n" +
-	"    trap - EXIT\n" +
-	"    exit \"$ret\"\n" +
-	"  }\n" +
-	"  trap __dump_at_exit EXIT\n" +
-	"\n" +
-	"  # load direnv libraries\n" +
-	"  for lib in \"$direnv_config_dir/lib/\"*.sh; do\n" +
-	"    # shellcheck disable=SC1090\n" +
-	"    source \"$lib\"\n" +
-	"  done\n" +
-	"\n" +
-	"  # load the global ~/.direnvrc if present\n" +
-	"  if [[ -f $direnv_config_dir/direnvrc ]]; then\n" +
-	"    # shellcheck disable=SC1090\n" +
-	"    source \"$direnv_config_dir/direnvrc\" >&2\n" +
-	"  elif [[ -f $HOME/.direnvrc ]]; then\n" +
-	"    # shellcheck disable=SC1090\n" +
-	"    source \"$HOME/.direnvrc\" >&2\n" +
-	"  fi\n" +
-	"\n" +
-	"  # and finally load the .envrc\n" +
-	"  \"$@\"\n" +
-	"}\n" +
-	""
+//go:embed stdlib.sh
+var StdLib string
```

Comment:
hey @zimbatm, the directive is right here (github was collapsing my changes here)